grakn-test/test-integration/src/test/java/ai/grakn/test/rule/DistributionContext.java

```diff
@@ -88,10 +92,7 @@ public SimpleURI uri(){
 
     @Override
     protected List<TestRule> testRules() {
-        return ImmutableList.of(
-                SessionContext.create(),
-                InMemoryRedisContext.create(redisPort)
-        );
+        return ImmutableList.of(session, redis);
```

Comment:
This is kind of funny when you really think about it :D

Reply:
Haha 😢